ports/glutin/window.rs

```diff
@@ -182,6 +182,8 @@ pub struct Window {
     mouse_down_point: Cell<Point2D<i32>>,
     event_queue: RefCell<Vec<WindowEvent>>,
 
+    browser_id: Cell<Option<BrowserId>>,
```

Comment:
Why is this an Option? Is it because of initalization? If so, can we add a comment to that effect?